### DISCUSSION TITLE
Is there any way to integrated with dmenu like rofi?

### DISCUSSION THREAD

**Discussion by jeffrey4l** [knqyf263/pet#298] [Ideas]
It will be great if pet can work with dmenu, like rofi

something like below

```bash
pet list --oneline | rofi -i -dmenu | awk -F' : ' '{print $2}' | tr -d '\n' | xsel -b -i
```

but the content is cut for `Command` and `Description`

**Comment by RamiAwar** [comment]
I think this is a fair request. I think it's better if Pet provided the most info (full one liner) and then the user adds truncate commands:

```pet list --oneline | cut -c 1-100```

**Comment by RamiAwar** [comment]
You can expect this merged today, maybe I'll do a new release later tonight as well.

**Reply by RamiAwar** [reply]
Fix is added to #301
